woocommerce/woocommerce-android
# [Manual Taxes M2] tax selector empty state

Closes: #9758 


### Description


This PR adds the Tax Rate Selector list empty state.

Design: ue2M9WUgbPdPk8Up5Sxzby-fi-365_7857

### Testing instructions


- Log in with a site that has not taxes set up
- Go to the Orders tab
- add a new order
- click on the Set New Tax rate button 
- make sure the empty state is shown 
- click on the button and make sure it takes you to Wp-Admin

### Images/gif


| Light | Dark |
| ---- | ---- |
| <img src="https://github.com/woocommerce/woocommerce-android/assets/30724184/061b56d8-9435-41e4-8e2b-0d839a398f2a" width="350"/> | <img src="https://github.com/woocommerce/woocommerce-android/assets/30724184/e8c85e17-b5a4-40c2-9dd6-88cd5ac94a4a" width="350"/> |
